src/olympia/editors/urls.py

```diff
@@ -35,7 +37,9 @@
         name='editors.queue_viewing'),
     url(r'^review_viewing$', views.review_viewing,
         name='editors.review_viewing'),
-    url(r'^review(?:-(?P<channel>listed|unlisted))?/%s$' % ADDON_ID,
+    # 'content' is not a channel, but is a special kind of review that we only
+    # do for listed add-ons, so we abuse the channel parameter to handle that.
```

Comment:
🙄 but okay.

Reply:
I'm not a big fan either, and I knew you'd disapprove :) I can try going back and finding a better way of doing this.

Reply:
Hmm, maybe put that in a separate view that has the same URL though (`/review/content/<id>/`) and just uses the review view for rendering?

Reply:
`/review/content/<id>/` will be problematic - you could have an add-on with the slug `content`.